Runtime/Native/XBOX/NativeClient.cs

```diff
@@ -0,0 +1,140 @@
+﻿#if UNITY_GAMECORE_XBOXSERIES
+using Backtrace.Unity.Interfaces;
+using Backtrace.Unity.Model;
+using Backtrace.Unity.Model.Breadcrumbs;
+using Backtrace.Unity.Runtime.Native.Base;
+using Backtrace.Unity.Types;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+
+[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Backtrace.Unity.Tests.Runtime")]
+namespace Backtrace.Unity.Runtime.Native.XBOX
+{
+
+    internal class NativeClient : NativeClientBase, INativeClient
+    {
+        [DllImport("breakpad_xbox_simplified_abi_mt.dll")]
```

Comment:
I am not. It's "simplified ABI" since the dll only exposes a few basic functions, C-style (undecorated ABI), which is different from the full Breakpad library.

I am open to changing the name.

Reply:
we can hide what is this because I'm not sure if this is important to share that this is c-style library. I think it's not from the end user perspective. How about: ` backtrace-unity-xbox` ?

Reply:
This library could be used from other game engines as well. I would rather not package the same library several times just by changing the name.